### ISSUE TITLE
2020-9-15

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#315]
701.2.Evaluation protocol; follow the same protocol in TransE; replace the tail t by every entity e in the knowledge graph; averaged rank, proportion of ranks not larger than 10; “filt”, remove those corrupted triplets which exist in either training, valid, or testing set before getting the rank of each testing triplet; (2020-9-7)
702.3.Implementation; directly copy experimental results of several baselines; use learning rate; margin; embedding dimension; weight C; batch size B; “unif” to denote the traditional way; “bern.” to denote replacing head or tail with different probabilities; (2020-9-8)
703.4.Table 3: Link prediction results; (2020-9-8)
704.5.Results; number of relations in WN18 is quite small so that it is acceptable to ignore the different types of relations; TransE is the runner up on FB15k; The imrovement of TransH over TransE on these relations are very promising; (2020-9-8)
705.Triplets Classification
706.1.confirm whether a given triplet (h; r; t) is correct or not, i.e., binary classification on a triplet; (2020-9-8)
707.Relational Fact Extraction from Text
708.1.knowledge graph embedding is able to score a candidate fact, without observing any evidence from external text corpus; combined the score from TransE (evidence from knowledge graphs) with the score from a text side extraction model (evidence from text corpus); improve relational fact extraction; (2020-9-8)
709.2.text side extraction model and knowledge graph embedding; (2020-9-8)
